pom.xml

```diff
@@ -24,7 +27,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-    <version.drools>8.26.0.Beta</version.drools>
+    <version.drools>8.29.0-SNAPSHOT</version.drools>
```

Comment:
can't we use a recent Final?

Reply:
Yes, for the moment being.
All the improvements in efesto are in the current snapshot, though....